src/loaders/abstract_loader.py

```diff
@@ -44,12 +44,17 @@ class AbstractVideoLoader(metaclass=ABCMeta):
     """
 
     def __init__(self, video_metadata: VideoMetaInfo, batch_size=1,
-                 skip_frames=0, offset=None, limit=None):
+                 skip_frames=0, offset=None, limit=None, threshold=0, 
```

Comment:
We don't want to change the abstract Loader. It will be better if you guys can add a new subclass with your updated parameters.